.github/workflows/unit.yaml

```diff
@@ -0,0 +1,15 @@
+name: ci
+on: [push]
```

Comment:
This is needed right now, but in the future, we may want to restrict _which_ PRs get run based on what changes are included in the PR. A common example is a PR that only changes markdown files (i.e. a docs-only PR) that running unit tests against may be a waste of CI resources.

Reply:
A concrete example that excludes certain paths: https://github.com/operator-framework/operator-lifecycle-manager/blob/master/.github/workflows/unit.yml#L6-L8.

Reply:
Hm, that's interesting. Looking into OLM, I noticed this:

```yaml
on:
  push:
    branches:
      - master
  pull_request:
    paths-ignore:
    - '**/*.md'
```

Its an easy enough fix, would there be a reason to not add this now?

Reply:
Nice! I'll go ahead and add that then.